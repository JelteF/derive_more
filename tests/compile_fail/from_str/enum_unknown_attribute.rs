#[derive(derive_more::FromStr)]
#[from_str(unknown = "unknown")]
pub enum Foo {
    Bar,
}

fn main() {}
