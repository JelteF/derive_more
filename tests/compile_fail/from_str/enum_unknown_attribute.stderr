error: unknown attribute argument, expected `rename_all = "..."`
 --> tests/compile_fail/from_str/enum_unknown_attribute.rs:2:12
  |
2 | #[from_str(unknown = "unknown")]
  |            ^^^^^^^
